fcgiwrap.c

```diff
@@ -578,6 +578,7 @@ static void handle_fcgi_request(void)
 			}
 
 			execl(filename, filename, (void *)NULL);
+			fprintf(stderr, "Error: %s\n", strerror(errno));
```

Comment:
I am not against removing the "Error", but cgi_error() function will also output "Cannot execute script ....", then it is kind of redundant to print "Failed to execute", but printing only the strerror(errno) without "Error" makes it a little confusing. Any idea on this?
@Lekensteyn